tools/prepare-commit.sh

```diff
@@ -102,6 +102,9 @@ EOT
 	fi
 done
 
+# Build
+./gradlew build
```

Comment:
That's an internal change I added to this pull-request to avoid adding wrong files, like I just did. You can ignore it.